com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/dao/ConfigurationEntryDao.java

```diff
@@ -51,14 +52,30 @@ public ConfigurationEntry update(long id, ConfigurationEntry entry) throws Confl
     public ConfigurationEntry find(long id) throws NotFoundException {
         return dao.find(id).toConfigurationEntry();
     }
+    
+    public List<ConfigurationEntry> find(String guid){
```

Comment:
Why is there a separate find method for the space ID? Why not add it to the other find method (the one with the million arguments)?